go/ct/evm_test.go

```diff
@@ -37,13 +37,15 @@ var evms = map[string]ct.Evm{
 func TestCt_ExplicitCases(t *testing.T) {
 	tests := map[string]Condition{
 		"jump_to_2^32": And(
+			RevisionBounds(R07_Istanbul, R10_London), // TODO: update to newer revisions as suppor is added
```

Comment:
Please use the [NewestSupportedRevision](https://github.com/Fantom-foundation/Tosca/blob/bc85fbcf8f913ead471a408e81f38295f5986ac3/go/ct/common/revisions.go#L34) constant here and below.